src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -8,26 +14,145 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Event;
 import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
 import java.util.Enumeration;
+import java.util.List;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** 
+ * Servlet that currently allows for retrieving info from url and put in 
+ *  calendar
+ */
 @WebServlet("/calculate-trip")
 public class TripServlet extends HttpServlet {
 
+  // time class constants
+  private static final int HALF_HOUR = 30;
+  private static final int NINETY_MINS = 90;
+
+  // event fields for entity
+  private static final String NAME = "name";
+  private static final String ADDRESS = "end-time";
+  private static final String DATE = "date";
+  private static final String START_TIME = "start-time";
+  private static final String TRAVEL_TIME = "travel-time";
+
+  
+
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+      throws IOException {
     response.setContentType("application/json;");
+    
+    // do get for events
+    eventDoGet(response);
+  }
+
+  // function to set time, need to do only once
+  public void setDateTime(String date) {
+    if (startDateTime == null) {
+      startDateTime = LocalDateTime.of(LocalDate.parse(date), LocalTime.of(10, 0));
```

Comment:
I'm seeing a compilation error on this line since startDateTime is no longer a class field. I think this method should have a LocalDateTime return type instead of void

Reply:
Done! I didn't catch that